app/client/src/components/shared/LocationSearch/index.js

```diff
@@ -473,7 +473,7 @@ function LocationSearch({ route, label }: Props) {
     setSuggestionsVisible(false);
     setCursor(-1);
 
-    newSearchTerm = newSearchTerm.replace(/[\n\r\t]/g, ' ');
+    newSearchTerm = newSearchTerm.replace(/[\n\r\t\/]/g, ' ');
```

Comment:
ESLint is telling me you don't need to escape the forward slash, since it's inside a character class (square brackets)

```suggestion
    newSearchTerm = newSearchTerm.replace(/[\n\r\t/]/g, ' ');
```

Reply:
Good catch. I removed the escape character.